control-plane/connect-inject/webhook/consul_dataplane_sidecar.go

```diff
@@ -376,10 +376,16 @@ func (w *MeshWebhook) getContainerSidecarArgs(namespace corev1.Namespace, mpi mu
 		args = append(args, fmt.Sprintf("-shutdown-grace-period-seconds=%d", shutdownGracePeriodSeconds))
 
 		gracefulShutdownPath := w.LifecycleConfig.GracefulShutdownPath(pod)
+		args = append(args, fmt.Sprintf("-graceful-shutdown-path=%s", gracefulShutdownPath))
+
+		startupGracePeriodSeconds, err := w.LifecycleConfig.StartupGracePeriodSeconds(pod)
 		if err != nil {
-			return nil, fmt.Errorf("unable to determine proxy lifecycle graceful shutdown path: %w", err)
```

Comment:
Just a note here: the `if err != nil` statement below `w.LifecycleConfig.GracefulShutdownPath(pod)` was useless as that method doesn't return 2 values. Thus I've removed it.